### PR TITLE
Update for newer node

### PR DESCRIPTION
**Pull request by marcandre**
Was there a particular reason for the upper bound on the version of Node?

Tests run fine on Node v10.9.0.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+yarn.lock
```
Comment (treycordova) [inline .gitignore]: What's your justification for ignoring `yarn.lock`?

**Review by treycordova — CHANGES_REQUESTED** [review]
Happy to accept this change, but I have one small comment.

File: .gitignore (hunk 1 above)
Comment (marcandre) [reply .gitignore]: Oh, I can remove this commit. It's simply that installing this in dev will create either a `yarn.lock`, or a `package.json.lock`. These aren't typically committed for libraries.

**Comment by marcandre** [comment]
Commit removed.

**Comment by marcandre** [comment]
Would you please merge this and make a new release?
We intend to use this within backyourstack

**Comment by marcandre** [comment]
Rebase branch. Lint issue fixed. Pinged `gemfile` lib author.

**Comment by treycordova** [comment]
Merged and published as 1.1.0. Happy coding!

**Comment by marcandre** [comment]
Thanks!

**Pull request merged**